package.json

```diff
@@ -31,28 +31,31 @@
   "dependencies": {
     "archiver": "^1.0.0",
     "colors": "0.6.0-1",
-    "connect-phonegap": "0.24.5",
-    "cordova": "6.4.0",
+    "connect-phonegap": "https://github.com/stevengill/connect-phonegap.git#177",
+    "cordova-create": "^1.0.1",
+    "cordova-common": "^2.0.0",
+    "elementtree": "^0.1.6",
```

Comment:
@stevengill didn't you say there was a breaking change in elementtree 0.1.7 that fucked shit up outta nowhere for you?